.github/workflows/update-binaries.yaml

```
@@ -1,5 +1,9 @@
 name: Update bin.walrus.site with latest binaries
 
+on:
+  repository_dispatch:
+    types: [update-sites-bins]
+
 on:
```

Nit: Can we merge these?
```suggestion
on:
  repository_dispatch:
    types: [update-sites-bins]
```